CMakeLists.txt

```diff
@@ -31,6 +32,7 @@ set(SOURCES_LIST
     strings/aho_corasick
     various/set_tricks
     various/compression
+    ../kactl/number-theory/ModSqrt
```

Comment:
Low priority now, but `SOURCES_LIST` should probably be split up into `nerdi` and `kactl` or something. The `..` make it look obscure.

Reply:
It's a hack for sure, but due to the time constraints I went with this.